protocol/avdl/gregor1/incoming.avdl

```diff
@@ -8,6 +8,7 @@ protocol incoming {
   SyncResult sync(UID uid, DeviceID deviceid, Time ctime);
   void consumeMessage(Message m);
   void consumePublishMessage(Message m);
+  void consumeMessageMulti(Message msg, array<UID> uids);
```

Comment:
Maybe just drop a small comment on this guy, since the behavior is slightly weird?